tensornetwork/backends/jax/jax_backend_test.py

```diff
@@ -530,3 +531,27 @@ def test_matrix_ops_raises(dtype, method):
   matrix = backend.randn((4, 3), dtype=dtype, seed=10)
   with pytest.raises(ValueError, match=r".*N\*N matrix.*"):
     getattr(backend, method)(matrix)
+
+
+def test_jit():
+  backend = jax_backend.JaxBackend()
+
+  def fun(x, A, y):
+    return jax.numpy.dot(x, jax.numpy.dot(A, y))
+
+  fun_jit = backend.jit(fun)
+  x = jax.numpy.array(np.random.rand(4))
+  y = jax.numpy.array(np.random.rand(4))
+  A = jax.numpy.array(np.random.rand(4, 4))
+
+  t1 = time.time()
+  res = fun_jit(x, A, y)
+  res.block_until_ready()
+  t2 = time.time()
+
+  res = fun_jit(x, A, y)
+  res.block_until_ready()
+  t3 = time.time()
+  dt2 = t3 - t2
+  dt1 = t2 - t1
+  assert dt2 < dt1
```

Comment:
Don't try to do timing tests here, its far too flaky. 

A simple test of "The original function and the jitted function return the same result" is good enough. Also, be sure to test with and without keyword arguments since we missed `*args` earlier.